hw3/output/HW3#2_Table.csv

```diff
@@ -0,0 +1,15 @@
+Cluster  Max Int
```

Comment:
What happens if you try to open this file in a spreadhseet? You may need to reformat this a bit. Also, what are the units for x, y and z?